plugins/modules/ovirt_host_storage_info.py

```diff
@@ -134,26 +142,20 @@ def main():
         # Get Host
         hosts_service = connection.system_service().hosts_service()
         host_id = get_id_by_name(hosts_service, module.params['host'])
-        storage_type = _get_storage_type(module.params)
         host_service = hosts_service.host_service(host_id)
 
-        if storage_type == 'iscsi':
+        if module.params.get('iscsi'):
```

Comment:
IMHO more readable to define before this line:
    iscsi = module.params.get('iscsi')
and use everywhere (5 places).
Can be done also for fcp for readability
    module.params.get('fcp')

Reply:
Yeah, will do